aspnetcore/fundamentals/http-context/samples/6.x/HttpContextInBackgroundThread/PeriodicBranchesLoggerService.cs

```diff
@@ -29,17 +30,20 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                 syncTokenSource.CancelAfter(TimeSpan.FromSeconds(30));
                 
                 var httpClient = _httpClientFactory.CreateClient("GitHub");
-                var httpResponseMessage = await httpClient.GetAsync("repos/dotnet/AspNetCore.Docs/branches", stoppingToken);
+                var httpResponseMessage = await httpClient.GetAsync("repos/dotnet/AspNetCore.Docs/branches",
+                                                                    stoppingToken);
 
                 if (httpResponseMessage.IsSuccessStatusCode)
                 {
                     await using var contentStream =
                         await httpResponseMessage.Content.ReadAsStreamAsync(stoppingToken);
 
                     // Sync the response with preferred datastore.
-                    var response = await JsonSerializer.DeserializeAsync<IEnumerable<GitHubBranch>>(contentStream, cancellationToken: stoppingToken);
+                    var response = await JsonSerializer.DeserializeAsync<
+                        IEnumerable<GitHubBranch>>(contentStream, cancellationToken: stoppingToken);
 
-                    _logger.LogInformation($"Branch sync successful! Response: {JsonSerializer.Serialize(response)}");
+                    _logger.LogInformation(
+                        $"Branch sync successful! Response: {JsonSerializer.Serialize(response)}");
                 }
```

Comment:
This should probably log something for unsuccessful status codes as well.

```suggestion
                }
                else
                {
                    _logger.LogError(1, $"Branch sync failed! HTTP status code: {httpResponseMessage.StatusCode}");
                }
```